doc/releases/reef.rst

```diff
@@ -6,6 +6,438 @@ Reef is the 18th stable release of Ceph. It is named after the reef squid
 (Sepioteuthis).
 
 
+v18.2.4 Reef
+============
+
+This is the fourth backport release in the Reef series. We recommend that all users update to this release.
+
+An early build of this release was accidentally exposed and packaged as 18.2.3 by the Debian project in April.
+That 18.2.3 release should not be used. The official release was re-tagged as v18.2.4 to avoid
+further confusion.
+
+v18.2.4 container images, now based on CentOS 9, may be incompatible on older kernels (e.g., Ubuntu 18.04) due
+to differences in thread creation methods. Users upgrading to v18.2.4 container images on older OS versions
+may encounter crashes during pthread_create. For workarounds, refer to the related tracker. However, we recommend
+upgrading your OS to avoid this unsupported combination.
+Related tracker: https://tracker.ceph.com/issues/66989
+
+Notable Changes
+---------------
+
+* RADOS: This release fixes a bug (https://tracker.ceph.com/issues/61948) where pre-reef clients were allowed
```

Comment:
Reef

Reply:
@anthonyeleven I'm not sure if you have the permissions, but feel free to push a small change like this directly to the PR.

Other bigger changes, like rewording a release note, can be left to the developers to review.